code/game/objects/items/storage/bags.dm

```diff
@@ -426,6 +426,7 @@
 	worn_icon_state = "biobag"
 	desc = "A bag for the safe transportation and disposal of biowaste and other biological materials."
 	resistance_flags = FLAMMABLE
+	slot_flags = ITEM_SLOT_BELT | ITEM_SLOT_POCKETS
```

Comment:
you may want to mention in the pr that you're letting bio bags also fit into pockets with this pr as well, making it up to 300 slime cores on a person at once

Reply:
or don't do this at all